File: 01.CSharpPartOne/05_ConditionalStatements/ConditionalStatements/08DigitAsWord/DigitAsWord.cs
using System;

/*Problem 8. Digit as Word

Write a program that asks for a digit (0-9), and depending on the input, shows the digit as a word (in English).
Print “not a digit” in case of invalid input.
Use a switch statement.
 */
class DigitAsword
{
    static void Main(string[] args)
    {
        Console.WriteLine("Enter number between 0 and 9:");
        double number = double.Parse(Console.ReadLine());

        if ((number < 0) || (number > 9) || ((number % 1) != 0))
        {
            number = 123437;
        }

        switch ((int)number)
        {
            case 0:
                Console.WriteLine("zero");
                break;
            case 1:
                Console.WriteLine("one");
                break;
            case 2:
                Console.WriteLine("two");
                break;
            case 3:
                Console.WriteLine("three");
                break;
            case 4:
                Console.WriteLine("four");
                break;
            case 5:
                Console.WriteLine("five");
                break;
            case 6:
                Console.WriteLine("six");
                break;
            case 7:
                Console.WriteLine("seven");
                break;
            case 8:
                Console.WriteLine("eight");
                break;
            case 9:
                Console.WriteLine("nine");
                break;
            default:
                Console.WriteLine("Not a digit.");
                break;
        }
    }
}

File: 06_JavaScript/03_ContitionalStatements/ConditionalStatements/05.DigitAsWord/DigitAsWord.js
var digit;
var testDigits = [2, 1, 0, 5, -0.1, 'hi', 9, 10];

for (var i = 0; i < testDigits.length; i++) {
    digit = testDigits[i];

    switch (digit) {
        case 0:
            console.log('zero');
            document.write('<p>zero</p>')
            break;
        case 1:
            console.log('one');
            document.write('<p>one</p>')
            break;
        case 2:
            console.log('two');
            document.write('<p>two</p>')
            break;
        case 3:
            console.log('three');
            document.write('<p>three</p>')
            break;
        case 4:
            console.log('four');
            document.write('<p>four</p>')
            break;
        case 5:
            console.log('five');
            document.write('<p>five</p>')
            break;
        case 6:
            console.log('six');
            document.write('<p>six</p>')
            break;
        case 7:
            console.log('seven');
            document.write('<p>seven</p>')
            break;
        case 8:
            console.log('eight');
            document.write('<p>eight</p>')
            break;
        case 9:
            console.log('nine');
            document.write('<p>nine</p>')
            break;
        default:
            console.log('not a digit');
            document.write('<p>not a digit</p>')
            break;
    }
}